java/download-preplanned-map-area/README.md

```diff
@@ -2,16 +2,18 @@
 
 Take a map offline using a preplanned map area.
 
-![Download preplanned map area app](download-preplanned-map-area.png)
+![Image of download preplanned map area](download-preplanned-map-area.png)
 
 ## Use case
 
 Generating offline maps on demand for a specific area can be time consuming for users and a processing load on the server. If areas of interest are known ahead of time, a web map author can pre-create packages for these areas. This way, the generation only needs to happen once, making the workflow more efficient for users and servers.
 
-An team of archaeologists could define preplanned map areas for dig sites which can be taken offline for field use.
+An archaeology team could define preplanned map areas for dig sites which can be taken offline for field use.
 
 ## How to use the sample
 
+Downloading tiles for offline use requires authentication with the web map's server. An [ArcGIS Online](www.arcgis.com) account is required to use this sample.
+
 Select a map area from the Preplanned Map Areas list. Click the Download button to download the selected area. The download progress will be shown in the Downloads list. When a download is complete, select it to display the offline map in the map view.
```

Comment:
I found that after the selected area has downloaded it automatically displayed on the mapview. After downloading multiple area, I could go back and select among them to display my selection.

Reply:
I'm confused, isn't that what this says?

Reply:
```suggestion
Select a map area from the Preplanned Map Areas list. Click the Download button to download the selected area. The download progress will be shown in the Downloads list. When a download is complete, it will automatically display in the map view. You can also select a previously downloaded area to display it in the map view.
```